composer.json

```diff
@@ -33,6 +33,7 @@
         "zendframework/zenddiagnostics": "^1.0"
     },
     "suggest": {
+        "sonata-project/admin-bundle": "^3.1",
```

Comment:
A suggest should not contain a version constraint. I know it's already done for other, we have to change it.

Instead, please add a relevant description and use the `conflict` section for version constraint.